pkg/registry/scripts/main.js

```diff
@@ -58,20 +60,17 @@
     ])
 
     .controller('MainCtrl', [
+        '$q',
         '$scope',
         '$location',
         '$rootScope',
         '$timeout',
         'kubeLoader',
-        'kubeSelect',
+        'registrySettings',
         'filterService',
         'cockpitKubeDiscover',
-        function($scope, $location, $rootScope, $timeout, loader, select, filter, discover) {
```

Comment:
Your suggestion discussed on chat was good. Like you said it doesn't need to be included in this pull request.